pkg/deployments/src/network.ts

```diff
@@ -39,3 +41,28 @@ export function getContractDeploymentTransactionHash(deployedAddress: string, ne
 
   return txHash;
 }
+
+export function saveContractDeploymentAddresses(task: Task): void {
+  if (task.network === 'hardhat') return;
+
+  const newEntries = Object.fromEntries(
+    Object.entries(task.output({ ensure: false }))
+      .filter(([name]) => name !== 'timestamp')
```

Comment:
:sadge:
we should remove these timestamp fields

Reply:
Yeah, I don't think we use them at all and we can look up the deployment tx in a pinch.